[Build] Build failing in some environments

Build is currently failing on CI server (hulk) due to:

```
> phantomjs-prebuilt@2.1.12 install /home/vwoeltje/tmp/build-failure/openmct/node_modules/phantomjs-prebuilt
> node install.js

module.js:327
    throw err;
    ^

Error: Cannot find module 'asn1'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/vwoeltje/tmp/build-failure/openmct/node_modules/request/node_modules/http-signature/node_modules/sshpk/lib/formats/pem.js:9:12)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

After subsequently running `npm install phantomjs-prebuilt` separately, `npm install` works as expected.

May be duplicate of #1174 (depending on root cause of that issue)

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "mkdirp": "^0.5.1",
     "moment": "^2.11.1",
     "node-bourbon": "^4.2.3",
-    "phantomjs-prebuilt": "^2.1.0",
+    "phantomjs-prebuilt": "2.1.11",
     "requirejs": "2.1.x",
     "split": "^1.0.0"
   },

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "mkdirp": "^0.5.1",
     "moment": "^2.11.1",
     "node-bourbon": "^4.2.3",
-    "phantomjs-prebuilt": "2.1.11",
+    "phantomjs-prebuilt": "2.1.11 || >2.1.12 <3.0.0",
     "requirejs": "2.1.x",
     "split": "^1.0.0"
   },

